doc/src/records/dhwsys.md

```diff
@@ -212,7 +388,7 @@ Hot water delivery temperature (at output of water heater(s) and at point of use
 
 **wsTSetPoint=*float***
 
-  Specifies the hot water setpoint temperature for all child DHWHEATERs.  Used only for HPWH-based DHWHEATERs (HPWH models tank temperatures and heating controls), otherwise has no effect.
```

Comment:
This shouldn't have changed. I think your spell checker doesn't recognize "setpoint". You should add it to your dictionary.